### PR TITLE
CORS-3617: Promote Azure CAPI as the default Install method

### PR DESCRIPTION
**Pull request by sadasu**

### CODE REVIEW

**Comment by sadasu** [comment]
[Azure Techpreview job results](https://sippy.dptools.openshift.org/sippy-ng/tests/4.17/analysis?test=install%20should%20succeed%3A%20infrastructure&filters=%7B%22items%22%3A%5B%7B%22columnField%22%3A%22name%22%2C%22operatorValue%22%3A%22equals%22%2C%22value%22%3A%22install%20should%20succeed%3A%20infrastructure%22%7D%2C%7B%22columnField%22%3A%22variants%22%2C%22operatorValue%22%3A%22contains%22%2C%22value%22%3A%22upgrade-minor%22%2C%22not%22%3Atrue%7D%2C%7B%22columnField%22%3A%22variants%22%2C%22operatorValue%22%3A%22contains%22%2C%22value%22%3A%22aggregated%22%2C%22not%22%3Atrue%7D%2C%7B%22columnField%22%3A%22variants%22%2C%22operatorValue%22%3A%22contains%22%2C%22value%22%3A%22never-stable%22%2C%22not%22%3Atrue%7D%2C%7B%22columnField%22%3A%22variants%22%2C%22operatorValue%22%3A%22contains%22%2C%22value%22%3A%22Platform%3Aazure%22%7D%2C%7B%22columnField%22%3A%22variants%22%2C%22operatorValue%22%3A%22contains%22%2C%22value%22%3A%22FeatureSet%3Atechpreview%22%7D%5D%2C%22linkOperator%22%3A%22and%22%7D)

**Comment by patrickdillon** [comment]
Techpreview Azure Infrastructure is [passing at 97%](https://sippy.dptools.openshift.org/sippy-ng/tests/4.17/details?filters=%257B%2522items%2522%253A%255B%257B%2522columnField%2522%253A%2522current_runs%2522%252C%2522operatorValue%2522%253A%2522%253E%253D%2522%252C%2522value%2522%253A%25227%2522%257D%252C%257B%2522columnField%2522%253A%2522variants%2522%252C%2522not%2522%253Afalse%252C%2522operatorValue%2522%253A%2522contains%2522%252C%2522value%2522%253A%2522Platform%253Aazure%2522%257D%252C%257B%2522columnField%2522%253A%2522variants%2522%252C%2522not%2522%253Afalse%252C%2522operatorValue%2522%253A%2522contains%2522%252C%2522value%2522%253A%2522FeatureSet%253Atechpreview%2522%257D%252C%257B%2522columnField%2522%253A%2522variants%2522%252C%2522operatorValue%2522%253A%2522contains%2522%252C%2522value%2522%253A%2522Topology%253Aha%2522%257D%252C%257B%2522id%2522%253A99%252C%2522columnField%2522%253A%2522name%2522%252C%2522operatorValue%2522%253A%2522contains%2522%252C%2522value%2522%253A%2522infrastructure%2522%257D%252C%257B%2522columnField%2522%253A%2522name%2522%252C%2522operatorValue%2522%253A%2522contains%2522%252C%2522value%2522%253A%2522install%2520should%2522%257D%255D%252C%2522linkOperator%2522%253A%2522and%2522%257D&period=default&sort=asc&sortField=current_pass_percentage&view=Passing)

Looking through the 4 failures, only one of those appears to be a CAPI-based failure. The rest are Terraform.

That seems to be roughly comparable to [default infrastructure provisioning which is at 97.6%](https://sippy.dptools.openshift.org/sippy-ng/tests/4.17/details?filters=%257B%2522items%2522%253A%255B%257B%2522columnField%2522%253A%2522current_runs%2522%252C%2522operatorValue%2522%253A%2522%253E%253D%2522%252C%2522value%2522%253A%25227%2522%257D%252C%257B%2522columnField%2522%253A%2522variants%2522%252C%2522not%2522%253Afalse%252C%2522operatorValue%2522%253A%2522contains%2522%252C%2522value%2522%253A%2522Platform%253Aazure%2522%257D%252C%257B%2522columnField%2522%253A%2522variants%2522%252C%2522not%2522%253Atrue%252C%2522operatorValue%2522%253A%2522contains%2522%252C%2522value%2522%253A%2522FeatureSet%253Atechpreview%2522%257D%252C%257B%2522columnField%2522%253A%2522variants%2522%252C%2522operatorValue%2522%253A%2522contains%2522%252C%2522value%2522%253A%2522Topology%253Aha%2522%257D%252C%257B%2522id%2522%253A99%252C%2522columnField%2522%253A%2522name%2522%252C%2522operatorValue%2522%253A%2522contains%2522%252C%2522value%2522%253A%2522infrastructure%2522%257D%252C%257B%2522columnField%2522%253A%2522name%2522%252C%2522operatorValue%2522%253A%2522contains%2522%252C%2522value%2522%253A%2522install%2520should%2522%257D%252C%257B%2522columnField%2522%253A%2522variants%2522%252C%2522operatorValue%2522%253A%2522contains%2522%252C%2522value%2522%253A%2522Architecture%253Aamd64%2522%257D%255D%252C%2522linkOperator%2522%253A%2522and%2522%257D&period=default&sort=asc&sortField=current_pass_percentage&view=Passing)

I don't suspect we're going to be able to get verify to pass because it looks like some operators are failing to come up reliably in TechPreview

File: features/features.go [hunk 1]
```diff
@@ -541,7 +541,7 @@ var (
 				reportProblemsToJiraComponent("Image Registry").
 				contactPerson("flavianmissi").
 				productScope(ocpSpecific).
-				enableIn(configv1.Default, configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
+				enableIn(configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
```
Comment (stbenjam) [inline features/features.go:544]: Was this intended? I think it's why integration tests are failing

**Comment by stbenjam** [comment]
As long as we understand why techpreview installs themselves are unreliable and there's open bugs for them, I think it's fine to override.  I agree infrastructure looks comparable between techpreview and non-techpreview.

File: features/features.go (hunk 1 above)
Comment (sadasu) [reply features/features.go]: No. Seems like a rebase issue from my end. Updated.

**Comment by patrickdillon** [comment]
/test verify

verify seems wedged, stuck in pending state

**Pull request merged**